#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-manpage.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Correct manpage section.

@DPATCH@

diff -Naur tack-1.06.orig/tack.1 tack-1.06/tack.1
--- tack-1.06.orig/tack.1	2006-04-22 22:26:55.000000000 +0000
+++ tack-1.06/tack.1	2007-10-12 19:04:43.000000000 +0000
@@ -27,7 +27,7 @@
 .\"***************************************************************************
 .\"
 .\" $Id: tack.1,v 1.3 2006/04/22 22:26:55 tom Exp $
-.TH tack 1M ""
+.TH tack 1 ""
 .ds n 5
 .ds d @TERMINFO@
 .SH NAME
